drivers/docker/driver_test.go

```diff
@@ -2381,36 +2402,6 @@ func TestDockerDriver_Entrypoint(t *testing.T) {
 	require.Equal(t, entrypoint, container.Config.Entrypoint, "Incorrect entrypoint ")
 }
 
-func TestDockerDriver_Kill(t *testing.T) {
```

Comment:
This test is very slightly different from `TestDockerDriver_Start_Kill_Wait` and other Kill tests.  Opted to remove it instead of having yet more flaky tests :(.  Happy to bring it back if we determine a value for it.